InfiniteRecharge/src/main/java/frc/robot/config/Config.java

```diff
@@ -1,54 +1,113 @@
-    // Motors
+    // Motor IDs
 
     // Shooter
     public int AZIMUTH_MOTOR_ID = 15;
     public int SHOOTER_MOTOR_ID = 13;
     public int INTAKE_MOTOR_ID = 8;
 
+    // Drive
+
     //////////////////////////////////////////////////////////////////////////////
     // Vision
 
     //////////////////////////////////////////////////////////////////////////////
-    // Stuff that makes this work
+    // Motor Configs
     public static class ShooterConfig {
         public MotorConfig azimuth = new MotorConfig();
         public MotorConfig intake = new MotorConfig();
         public MotorConfig shooter = new MotorConfig();
     }
 
+    public static class DriveConfig {
+        public int MOTORS_PER_SIDE = 2;
+        public MotorConfig leftMotors[] = new MotorConfig[MOTORS_PER_SIDE];
+        public MotorConfig rightMotors[] = new MotorConfig[MOTORS_PER_SIDE];
+
+        public int[] leftIDs = { 1, 2 };
+        public int[] rightIDs = { 3, 4 };
+
+        public MotorConfig leftLeader = leftMotors[0];
+        public MotorConfig rightLeader = rightMotors[0];
+
+        public DriveConfig() {
+            for (int i = 0; i < MOTORS_PER_SIDE; i++) {
+                leftMotors[i] = new MotorConfig();
+                leftMotors[i].id = leftIDs[i];
+
+                rightMotors[i] = new MotorConfig();
+                rightMotors[i].id = rightIDs[i];
+
+                if (i > 0) {
+                    leftMotors[i].followingID = leftIDs[0];
+                    rightMotors[i].followingID = rightIDs[0];
+                }
+            }
+            leftLeader = leftMotors[0];
+            rightLeader = rightMotors[0];
+        }
+    }
+
     public ShooterConfig shooter = new ShooterConfig();
+    public DriveConfig drive = new DriveConfig();
 
     public Config() {
 
-    //////////////////////////////////////////////////////////////////////////////
-    // IDs (Again)
+        //////////////////////////////////////////////////////////////////////////////
+        // IDs (Again)
         shooter.azimuth.id = AZIMUTH_MOTOR_ID;
         shooter.intake.id = INTAKE_MOTOR_ID;
         shooter.shooter.id = SHOOTER_MOTOR_ID;
 
-    //////////////////////////////////////////////////////////////////////////////
-    // PIDFs
-        shooter.azimuth.positionPIDF = new PIDF(
-        1, // P
-        1, // I
-        1, // D
-        1 /// F
+        //////////////////////////////////////////////////////////////////////////////
+        // PIDFs
+
+        // Shooter
+        shooter.azimuth.positionPIDF = new PIDF(//
+                0, // P
+                0, // I
+                0, // D
+                0 /// F
+        );
+        shooter.shooter.velocityPIDF = new PIDF(//
+                0, // P
+                0, // I
+                0, // D
+                0 /// F
+        );
+
+        // Drive
+        drive.leftLeader.velocityPIDF = new PIDF(//
+                0.1, // P

```

Comment:
This was good for testing, but we should set these to 0 until we have actual constants.